### PR TITLE
Help/About: Update font sizes and spacing on About section.

### PR DESCRIPTION
**Pull request by ryelle**
Trac ticket: https://core.trac.wordpress.org/ticket/55434

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

**Comment by ryelle** [comment]
Updated headers for the other About sections:

Freedoms


Privacy


~ETA: Just realized I did the credits page incorrectly, fixing~

Credits

File: src/wp-admin/about.php [hunk 1]
```diff
@@ -88,16 +84,16 @@
...
 			</div>
 		</div>
 
 		<div class="about__section has-1-column">
-			<div class="column about__image is-vertically-aligned-center">
+			<div class="column about__image is-vertically-aligned-top">
 				<img src="https://make.wordpress.org/core/files/2022/05/about-60-templates.png" alt="" />
 			</div>
-			<div class="column">
+			<div class="column" style="padding-bottom:0">
```
Comment (peterwilsoncc) [inline src/wp-admin/about.php:96]: Given this is used a few times, it it possible to add an `is-padding...` helper class to the admin css.

Nit, not a blocker.

File: src/wp-admin/about.php [hunk 2]
```diff
@@ -56,10 +54,8 @@
 			</div>
 		</div>
 
-		<hr />
-
 		<div class="about__section has-2-columns">
-			<div class="column about__image is-vertically-aligned-center">
+			<div class="column about__image is-vertically-aligned-top">
 				<img src="https://make.wordpress.org/core/files/2022/05/about-60-writing-improvements.png" alt="" />
```
Comment (peterwilsoncc) [inline src/wp-admin/about.php]: Excuse my ignorance, but when do these files typically get uploaded to the CDN?

**Review by peterwilsoncc — APPROVED** [review]
Looks good to me testing in the browser.

Testing notes
✅ Behaves responsively 
✅ text size, spacing increased
✅ SVG images reasonably well compressed ([SVGOMG](https://jakearchibald.github.io/svgomg/) could save 50 bytes of each but I suspect that was rounding numbers down)

File: src/wp-admin/about.php (hunk 2 above)
Comment (ryelle) [reply src/wp-admin/about.php]: Thanks for the reminder, that needs to happen today.

File: src/wp-admin/about.php (hunk 1 above)
Comment (ryelle) [reply src/wp-admin/about.php:96]: I don't think this particular style would be used in future about pages, so I'd rather not introduce a new class for it.

**Comment by ryelle** [comment]
Updated the image source to use the s.w.org CDN 👍🏻 

> SVG images reasonably well compressed ([SVGOMG](https://jakearchibald.github.io/svgomg/) could save 50 bytes of each but I suspect that was rounding numbers down)

They've all been through SVGOMG already - the biggest change was removing `focusable`, which is technically not necessary but I'm adding it to these SVGs to keep it consistent with the change to the "6.0" SVG in #2683.

**Review by SergeyBiryukov — APPROVED** [review]

**Comment by ryelle** [comment]
Committed in [r53382](https://core.trac.wordpress.org/changeset/53382).

**Pull request closed without merging**